sqlglot/dialects/doris.py

```diff
@@ -20,6 +20,12 @@ def _lag_lead_sql(self, expression: exp.Lag | exp.Lead) -> str:
     )
 
 
+def _jsonb_extract_scalar_sql(expression: exp.JSONBExtractScalar) -> str:
+    path = expression.this
+    path_elements = path.strip("{}").split(",")
+    return f"'$.{'.'.join(path_elements)}'"
```

Comment:
What is this? It looks hacky

Reply:
ago , it is 
`'{A,B,C}'`
i want to it be 
`'$.A.B.C'` , so , I used the above method to pass strings